main.cpp

```diff
@@ -84,42 +102,88 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
 /*
  1)
  */
+void takeASeat(bool isFront, int maximumWeightLbs)
+{
+    ignoreUnused(isFront, maximumWeightLbs);
+}
 
 /*
  2)
  */
+int yenToDollars(int yen)
+{
+    ignoreUnused(yen);
+    return {};
+}
 
 /*
  3)
  */
+int findArea(int width = 10, int height = 10)
+{
+    ignoreUnused(width, height);
+    return {};
+}
 
 /*
  4)
  */
+bool determineResult(bool resultA = false, bool resultB = false)
+{
+    ignoreUnused(resultA, resultB);
+    return {};
+}
 
 /*
  5)
  */
+void mainMenu(bool isFront)
```

Comment:
This function name does not conform to the programming guideline:
```
Name Things what they are.
Name Actions based on what they do
```
The name you chose reflects a property (noun) and does not contain a verb, which would reflect an action.
Functions perform work, so their names must contain a verb which describes the work being performed.
Examples of function names that contain verbs:
- `computeAValue()`
- `estimateDistanceTraveled()`
- `performDiagnostics()`
- [ ] revise